webapp/Shared/Footer.razor

```diff
@@ -89,21 +55,14 @@
                 <div class="row">
                     <div class="col-12">
                         <div class="content">
-                            <ul class="footer-bottom-links">
-                                <li><a href="javascript:void(0)">Terms of use</a></li>
-                                <li><a href="javascript:void(0)"> Privacy Policy</a></li>
-                                <li><a href="javascript:void(0)">Advanced Search</a></li>
-                                <li><a href="javascript:void(0)">Site Map</a></li>
-                                <li><a href="javascript:void(0)">Information</a></li>
-                            </ul>
-                            <p class="copyright-text">Designed and Developed by <a href="https://graygrids.com/"
-                                    rel="nofollow" target="_blank">GrayGrids</a>
+                            <p class="copyright-text">Developed with ❤️ by <a href="http://ilmhub.uz/"
+                                    rel="nofollow" target="_blank">Ilmhub Team</a>
                             </p>
                             <ul class="footer-social">
-                                <li><a href="javascript:void(0)"><i class="lni lni-facebook-filled"></i></a></li>
-                                <li><a href="javascript:void(0)"><i class="lni lni-twitter-original"></i></a></li>
-                                <li><a href="javascript:void(0)"><i class="lni lni-youtube"></i></a></li>
-                                <li><a href="javascript:void(0)"><i class="lni lni-linkedin-original"></i></a></li>
+                                <li><a href="#"><i class="lni lni-telegram-original"></i></a></li>
+                                <li><a href="#"><i class="lni lni-facebook-filled"></i></a></li>
+                                <li><a href="#"><i class="lni lni-instagram-filled"></i></a></li>
+                                <li><a href="#"><i class="lni lni-youtube"></i></a></li>
```

Comment:
Shu joydagi linlarni to'ldirib qo'yish uchun Issue create qilib qo'ying. Esdan chiqadi